modules/programs/khal.nix

```diff
@@ -37,7 +37,7 @@ in
     [
     (generators.toINI {} {
       default = {
-        default_calendar = head (mapAttrsToList (n: v: baseNameOf v.vdirsyncer.local.path) (filterAttrs (_: a: a.primary) khalAccounts));
+        default_calendar = head (mapAttrsToList (n: v: baseNameOf v.path) (filterAttrs (_: a: a.primary) khalAccounts));
```

Comment:
This line fails with the somewhat cryptic “list index 0 is out of bounds” if `khal` isn’t enabled for any account.